search-app/src/cmr/search/results_handlers/stac_results_handler.clj

```diff
@@ -218,3 +223,63 @@
 (defmethod qs/single-result->response [:granule :stac]
   [context query results]
   (single-result->response context query results))
+
+(defn- get-fc-links
+  "Returns the links for feature collection"
+  [context query hits]
+  (let [{:keys [page-size offset]} query
+        page-num (if offset
+                   (inc (quot offset page-size))
+                   1)
+        item-count (if offset
+                     (+ offset page-size)
```

Comment:
I don't understand this - is seems wrong - is the offset the number of items per page?  If so, shouldn't this be multiplication and not addition?  Or is offset the total number of items seen so far?  Then adding page seems wrong.  If offset is 100 and page-size is 15 then the item-count is 115.  first page - item-count grows by 1, second page, it grows by 2, and so on.

Reply:
We are using the current page-num used by STAC request to figure out if there is a previous page and next page. So if the total item returned at this page is greater than the total match or the maximum items allowed, there won't be a next page.

Reply:
Thanks.  I got it now.